napari/layers/image/_image_constants.py

```diff
@@ -88,3 +88,15 @@ class ImageRendering(StringEnum):
     MINIP = auto()
     ATTENUATED_MIP = auto()
     AVERAGE = auto()
+
+
+class VolumeDepiction(StringEnum):
+    """Depiction: 3D depiction mode for images.
+
+    Selects a preset depiction mode in vispy
+            * volume: images are rendered as 3D volumes.
+            * plane: images are rendered as 2D planes embedded in 3D.
+    """
+
+    VOLUME = auto()
+    PLANE = auto()
```

Comment:
Should this `StringEnum` (and other in the same file) have a translation like other layers do? e.g: 
https://github.com/napari/napari/blob/a9c1c59b30565b1b56cc6293ccd84e9856278df7/napari/layers/points/_points_constants.py#L83-L100

Reply:
good catch, thanks! will fix